src/providers/ldap/sdap.c

```diff
@@ -903,7 +906,8 @@ errno_t setup_tls_config(struct dp_option *basic_opts)
         ret = ldap_set_option(NULL, LDAP_OPT_X_TLS_CACERTFILE, tls_opt);
         if (ret != LDAP_OPT_SUCCESS) {
             DEBUG(SSSDBG_CRIT_FAILURE,
-                  "ldap_set_option failed: %s\n", sss_ldap_err2string(ret));
+                  "ldap_set_option failed(cacertfile): %s\n",
```

Comment:
Copy paste issue, it is:
"ldap_set_option failed(cacertfile)"
but I guess it should be:
"ldap_set_option(cacertfile) failed"

Reply:
Well, my intention was to make a set of similar error messages different to be able to figure out what exactly failed. Both forms work.
If you prefer the latter, I don't mind.
Changed.